lib/drivers/medtronic/medtronicDriver.js

```diff
@@ -1586,11 +1586,10 @@ module.exports = function (config) {
     cleanup: function (progress, data, cb) {
       debug('in cleanup');
 
-      /* TODO: put this back in when we have new sound/earcon
       if(isBrowser) {
-        var snd = new Audio('../../../sounds/underwater.wav');
+        var snd = new Audio('../../../sounds/UploadSucceeded.wav');
```

Comment:
Uhm, I think this is duplicate now. IIRC the idea was to either enable it just for Medtronic Direct, or for everything  (in DriverManager).